hedy.py

```diff
@@ -2311,11 +2339,11 @@ def if_pressed():
                 points = points + 1
          """
         lines = lines if isinstance(lines, list) else [lines]
-        variables = [e.name for e in self.lookup.get_only_vars()]
-        if variables and lines:
-            variables = sorted(list(set(variables)))
-            global_vars = f"global {', '.join(variables)}"
-            lines.insert(0, global_vars)
+        # variables = [e.name for e in self.lookup.get_only_vars()]
```

Comment:
I guess we can remove it?

Reply:
oh yes, you are right, I forgot to refactor this!

Reply:
wait! how did it get merged when there are conversations to be resolved :(